Cargo.toml

```diff
@@ -64,8 +64,8 @@ windows-targets = "0.52"
 
 # wasm_js
 [target.'cfg(all(getrandom_backend = "wasm_js", target_arch = "wasm32", any(target_os = "unknown", target_os = "none")))'.dependencies]
-wasm-bindgen = { version = "0.2.96", default-features = false }
-js-sys = { version = "0.3.73", default-features = false }
+wasm-bindgen = { version = "0.2.98", default-features = false }
+js-sys = { version = "0.3.75", default-features = false }
```

Comment:
IIUC `js-sys` is used only in the atomic backend.

Reply:
Done.